app/styles/_variables.scss

```diff
@@ -91,7 +91,7 @@ $input-border-radius: 2px;
 
 $input-background-color-default: $color-white;
 
-$input-text-font-size-default: $font-size-body-20;
+$input-text-font-size-default: 16px;
```

Comment:
Is it worthwhile to add a comment here describing why this changed? From the looks of it, seems like this isn't well documented for iOS webviews.